includes/data-port/class-sensei-data-port-job.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * File containing the Sensei_Data_Port_Job class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Superclass of data import/export jobs. It provides basic functionality like logging, maintaining state, cleanup and
+ * running data port tasks which are registered by subclasses.
+ */
+abstract class Sensei_Data_Port_Job implements Sensei_Background_Job_Interface, JsonSerializable {
+	const OPTION_PREFIX = 'sensei-tools-';
```

Comment:
Maybe for now since this is just in this class we can name it `sensei-data-port-`

Reply:
Makes sense! 13f133f9a715848f415b95590078e0a0a878f1d0